smart-cards/src/main/java/gov/va/api/health/smartcards/patient/PatientController.java

```diff
@@ -95,28 +130,72 @@ void initDirectFieldAccess(DataBinder dataBinder) {
       @Valid @RequestBody Parameters parameters,
       @RequestHeader(name = "Authorization", required = false) String authorization) {
     checkState(!StringUtils.isEmpty(id), "id is required");
-    var credentialTypes = validateCredentialType(parameters);
+    var credentialTypes = getCredentialTypes(parameters);
+    validateCredentialTypes(credentialTypes);
     Patient.Bundle patients = fhirClient.patientBundle(id, authorization);
     Patient patient = getPatientFromBundle(patients, id);
     Immunization.Bundle immunizations = fhirClient.immunizationBundle(patient, authorization);
```

Comment:
Unrelated, but as some clean up... it would be cleaner for `immunizationBundle` to accept just patient ICN as the first argument instead of the whole patient.